src/helpers/debugDeep.ts

```diff
@@ -0,0 +1,21 @@
+import type { ReactTestRendererJSON } from 'react-test-renderer';
+import format from './format';
+
+/**
+ * Log pretty-printed deep test component instance
+ */
+export default function debugDeep(
+  instance: ReactTestRendererJSON | ReactTestRendererJSON[],
+  message?: string
+) {
+  const instances = Array.isArray(instance) ? instance : [instance];
```

Comment:
I wonder if `format` (and underlying `prettyFormat`) could actually handle array version. If so then maybe we should just pass it there. wdyt?

Reply:
`prettyFormat` takes `unknown` as a parameter, so will follow your suggestion